sycl/include/CL/sycl/backend/cuda_definitions.hpp

```diff
@@ -0,0 +1,24 @@
+//==------------ cuda_definitions.hpp - SYCL CUDA backend ------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
```

Comment:
It's being used by unittests and I don't think they have access to the source directory. I don't see a reason why it shouldn't live in a detail/backend folder however.

Reply:
They should have.
Tagging @alexbatashev

Reply:
@steffenlarsen they do have. See: https://github.com/intel/llvm/blob/3baec1873322778f88446cd22a67b6594cf57968/sycl/unittests/CMakeLists.txt#L27

Reply:
Looking at it again cuda_definitions.hpp is used is in pi_cuda.cpp, which doesn't access any files in the source directory.

Reply:
@steffenlarsen would it make sense to move the whole plugins dir inside the source directory?

Reply:
Possibly, though I think that should be something considered for a separate PR. I suggest I move this to include/CL/sycl/detail/backend/cuda_definitions.hpp for now and then it can be moved later if a decision is made to move plugin into source.